### PR TITLE
Save unlinked local files dialog prefs

### LINKED ISSUE #11439: Unlinked Local File dialog: remembering options
When importing files into JabRef, I'm almost always using the `Lookup -> Search for unlinked local files` dialog. It has a set of default parameters which may be useful for first importing your library, but to add only my most recent downloads I have to change every option, every time.  I'm always changing it to the same search values, though, so it would be extremely useful to have a way for Jabref to **remember the last-used parameters for the search dialog box**. 

I think the way this could be implemented without messing with anyone who prefers the functionality would be:
- Add a checkbox to the "Search for unlinked local files" dialog box to remember the last used parameters for the search; or
- Add an option in the "Linked files" section of the preferences, for "default unlinked file search parameters".
- 
I think the first option is preferable, as it makes it easiest for a new user to see that this option exists.

### PR DESCRIPTION
**Pull request by kataramarina**
Added new check in the dialog that allows to save selected options of the dialog through the work and in the propgram preferences.
image

Closes #11439

### Mandatory checks

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml
Comment (koppor) [inline src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml:67]: Just always do that. No need to explicitly do that.

File: src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java
Comment (koppor) [inline src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java:177]:
```suggestion
```

**Review by koppor — CHANGES_REQUESTED** [review]
In general, looks good.

Please just remove the option to enable storing --> It should always be stored. (I know, the issue asked otherwise, that slipped through while making it a good first issue)

File: src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java
Comment (kataramarina) [reply src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogView.java]: Removed commented code

File: src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml
Comment (kataramarina) [reply src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialog.fxml]: Removed checkbox from dialog and all related to this checkbox stuff

File: src/main/java/org/jabref/gui/externalfiles/ExternalFileSorter.java
Comment (kataramarina) [reply src/main/java/org/jabref/gui/externalfiles/ExternalFileSorter.java]: Removed comment

**Review by koppor — APPROVED** [review]
Looks good to me! PR awaits a second review, since I am not fully into the preferences handling.

**Review by Siedlerchr — APPROVED** [review]
Thanks a lot for your contribution! lgtm

**Comment by Siedlerchr** [comment]
@kataramarina  For your info, there was a little bug which resulted in an exception which prevented the opening of the dialog

**Pull request merged**
